xcm/xcm-executor/src/lib.rs

```diff
@@ -990,6 +992,17 @@ impl<Config: config::Config> XcmExecutor<Config> {
 		Ok((asset, reanchor_context))
 	}
 
+	fn try_reanchor_multilocation(
+		location: MultiLocation,
+		destination: &MultiLocation,
+	) -> Result<(MultiLocation, InteriorMultiLocation), XcmError> {
```

Comment:
I see why you introduced `Reanchorable`; let's use that after this PR gets merged.